aws/lambda/python/s3_cleanup.py

```
@@ -0,0 +1,58 @@
+    sys.exit(1)
+
+bucket_name = sys.argv[1]
+s3 = boto3.client("s3")
+
+lifecycle_config = {
+    "Rules": [
+        {
+            "ID": "Delete objects after 10 minutes",
+            "Status": "Enabled",
+            "Expiration": {
+                "Days": 1
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)